tracepoint_decode/src/charconv.rs

```diff
@@ -18,11 +18,6 @@ fn str_from_validated_utf8(valid_utf8: &[u8]) -> &str {
     return unsafe { str::from_utf8_unchecked(valid_utf8) };
 }
 
```

Comment:
Note: `char_from_u32` moved to `writers` since that's the only place that uses it.